.github/workflows/publish-release.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: az storage blob download --file ${{env.VSIX_NAME_TO_PUBLISH}} --account-name pvsc --container-name ${{env.BLOB_CONTAINER_NAME}} --name ${{ env.VSIX_NAME_UPLOADED_TO_BLOB }}
 
       - name: Publish
-        run: vsce_ publish --packagePath ${{env.VSIX_NAME_TO_PUBLISH}} --pat ${{secrets.VSCE_TOKEN}} --noVerify
+        run: vsce publish --packagePath ${{env.VSIX_NAME_TO_PUBLISH}} --pat ${{secrets.VSCE_TOKEN}} --noVerify
```

Comment:
Not sure we want this yet. Are we planning on publishing automatically from the pipelines?
This command line will publish directly to the marketplace.
This used to be done when we were in Azure Pipelines, when we could manually approve this stage.
However in github my understanding is that there's no approval stage, hence any commit pushed to releaes will end up pushing a new VSIX to the marketplace.

Reply:
Isn't this what we wanted with https://github.com/microsoft/vscode-jupyter/pull/4789 ?